extensions/wikia/WikiaMobile/WikiaMobileService.class.php

```diff
@@ -216,17 +216,21 @@ private function handleContent($content = ''){
 		wfProfileOut( __METHOD__ );
 	}
 
+	private function isArticleView() {
+		$action = $this->wg->Request->getVal( 'action', 'view' );
+		$nameSpace = $this->wg->Title->getNamespace();;
+
+		return ( ( $action == 'view' || $action == 'ajax' ) &&
+			$this->wg->Title->getArticleId() != 0 &&
+			( $nameSpace !== 2 && $nameSpace !== 500 ) // skip user profile and user blog pages
```

Comment:
use variables defined in /includes/Defines so NS_USER although I couldn't find const for user blog page